fast_barcode_scanner/lib/src/camera_controller.dart

```diff
@@ -113,6 +125,38 @@ class _CameraController implements CameraController {
   @override
   final events = ValueNotifier(ScannerEvent.uninitialized);
 
+  final _scannedCodeSubject = PublishSubject<ScannedBarcodes>();
+
+  @override
+  Stream<List<Barcode>> get scannedCodes {
+    // The general strategy here is to provide a real-time stream of codes that are currently visible by the camera
+    // 100ms seems to be enough time. The key is to set a timeout that is longer than the rate at which the codes are scanned
+    const timeout = Duration(milliseconds: 250);
+
+    // this silencer stream allows us to clear the stream when codes are no longer visible
```

Comment:
I am unsure if this justifies depending on the rxdart library.
Keeping the plugin as lean as possible should be the goal.

Reply:
@jhoogstraat I really like rxdart. It has a nice API and makes some things easier. But I understand what you are saying and I agree that if there's a reasonable way to remove a dependency, it makes maintenance much easier. I removed the dependency and accomplished this task another way. Thanks!